src/components/fieldHelpers/label.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from "react"
+
+interface Props {
+  label?: string
+  required?: boolean
+  htmlFor?: string
```

Comment:
Can we call it `fieldName` or something more descriptive?

Reply:
Yes definitely !